style.css

```diff
@@ -337,4 +345,39 @@ option{
 		text-align: left;
 		padding-left: 2rem;		
 	}
+	.row{
+		display: flex;
+		justify-content: center;
+		flex-direction: column;
+		width:114%;
+	}
+	.section-head{
+		height: 70rem;
+	}
+	.button-border{
+		margin-top: 8rem;
+		margin-left: -20rem;
```

Comment:
What does`-20rem` do here?

Reply:
Positions play  button to top left when screen is below 992px

Reply:
Okay let's catch up on Monday about this because there are better ways of positioning elements